test/spmd/test_spmd_lowering_context.py

```diff
@@ -27,6 +27,7 @@ def _get_computation_hlo_txt(self, ctx):
     return xb.get_computation_hlo(comp)
```

Comment:
fyi: https://github.com/pytorch/xla/pull/8478/files is merged so this change isn't needed to make TPU pass now